### PR TITLE
Adding the Diet Cookie.

### PR DESCRIPTION
**Pull request by LinoxGH**
- Adding a new food called "Diet Cookie".
- Adding a recipe for Magic Workbench, containing of an elytra scale and a cookie.
- Levitates the player for 3 seconds.
- Adding a new research called "Diet Cookie" costing 3 levels.

### CODE REVIEW

**Comment by TheBusyBiscuit** [comment]
Okay, this is extremely funny XD

File: src/me/mrCookieSlime/Slimefun/Lists/SlimefunItems.java [hunk 1]
```diff
@@ -98,6 +98,7 @@ private SlimefunItems() {}
 	
 	/*		 Food 		*/
 	public static final ItemStack FORTUNE_COOKIE = new CustomItem(Material.COOKIE, "&6Fortune Cookie", "", "&a&oTells you stuff about your Future :o");
+	public static final ItemStack DIET_COOKIE = new CustomItem(Material.COOKIE, "&6Flying Cookie", "", "&a&oA boring diet cookie.");
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Lists/SlimefunItems.java]: Better idea:

Keep the name "&6Diet Cookie" and change the lore to "&a&oA cookie so light, it makes you levitate!"

Comment (LinoxGH) [reply src/me/mrCookieSlime/Slimefun/Lists/SlimefunItems.java]: But I wanted the levitation effect a bit hidden. To create a surprising effect on the player

Comment (TheBusyBiscuit) [reply src/me/mrCookieSlime/Slimefun/Lists/SlimefunItems.java]: Well, "Flying Cookie" is a bit misleading though, players could expect they will fly and then just levitate.

Comment (TheBusyBiscuit) [reply src/me/mrCookieSlime/Slimefun/Lists/SlimefunItems.java]: How about this lore?

"&eA very &olightweight &r&ecookie"

And the name "&6Diet Cookie"

Comment (LinoxGH) [reply src/me/mrCookieSlime/Slimefun/Lists/SlimefunItems.java]: that sounds good

P.S
I originally made the cookie with the name FLYING_COOKIE but then I changed it to DIET_COOKIE I guess I just forgot to change the name part lol

**Pull request merged**